.github/workflows/smoke-test.yaml

```
@@ -3,63 +3,85 @@
-    paths-ignore:
-      - 'docs/**'
   merge_group:
     branches:
       - dev
 
 jobs:
+  skip_if_only_docs:
+    runs-on: ubuntu-latest
+    outputs:
+      only_docs: ${{ steps.changes.outputs.only_docs }}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)